ui/pools/PoolsTableItem.tsx

```diff
@@ -0,0 +1,59 @@
+import { Flex, Box, Td, Tr, Skeleton, Text } from '@chakra-ui/react';
+import React from 'react';
+
+import type { Pool } from 'types/api/pools';
+
+import AddressEntity from 'ui/shared/entities/address/AddressEntity';
+import PoolEntity from 'ui/shared/entities/pool/PoolEntity';
+
+type Props = {
+  item: Pool;
+  index: number;
+  page: number;
+  isLoading?: boolean;
+};
+
+const PAGE_SIZE = 50;
+
+const PoolsTableItem = ({
+  item,
+  page,
+  index,
+  isLoading,
+}: Props) => {
+  return (
+    <Tr>
+      <Td>
+        <Flex gap={ 2 } alignItems="start">
+          <Skeleton isLoaded={ !isLoading }>
+            <Text px={ 2 }>{ index + 1 + (page - 1) * PAGE_SIZE }</Text>
```

Comment:
Perhaps we can create a utility function to calculate this position information and reuse it on the Tokens page as well.